### ISSUE TITLE
Top and bottom banners being off center

### ISSUE THREAD

**Issue by thisisevanfox** [thisisevanfox/nhl-my-team-ios-widget#1]
Device: XS Max

![9rYaUYJ](https://user-images.githubusercontent.com/43059689/104107110-50ec7300-52ba-11eb-9357-d7b880299c2d.jpg)
![104061048-76478700-51bd-11eb-9a74-b8553f5e4ee3](https://user-images.githubusercontent.com/43059689/104107117-5f3a8f00-52ba-11eb-904d-2243f0007596.jpg)

**Comment by jrkmtf** [comment]
I’m having the same issue - same phone too, XS Max.

**Comment by thisisevanfox** [comment]
> 
> 
> I’m having the same issue - same phone too, XS Max.

Yes it's a bug with big screen resolution. With my iPhone 7 it works perfect. I hope to fix this soon.

**Comment by thisisevanfox** [comment]
Issue also occurs on iPhone 11. 
Reported on twitter.

**Comment by gabriochsky** [comment]
Hi there, late to the party but I’m also on iPhone 11, made some tweaks in the length, corner radius and paddings of the stacks

 ![IMG_0598](https://github.com/thisisevanfox/nhl-my-team-ios-widget/assets/150813554/fe06f360-df3f-41b9-8ea4-4cce49eb17d7)

**Comment by thisisevanfox** [comment]
Hi @gabriochsky, which changes did you made? Would you like to create a pull request or send me the changes, so that I can include them in the widget?

Regards,
Evan

**Comment by gabriochsky** [comment]
Hi @thisisevanfox, can't pull request or write to you but this is what I did, probably changed some unnecessary things on the small widget too but I did it on my phone in the metro this morning tbh. At least it works, not like the Habs tonight!
[nhl.txt](https://github.com/thisisevanfox/nhl-my-team-ios-widget/files/13359919/nhl.txt)

**Comment by thisisevanfox** [comment]
Hi @gabriochsky, thank you for your effort and providing the file. I've just tried it on my iPhone 13, and unfortunately the stacks are now displayed too wide. I'm still trying things out and hope to be able to solve the problem for all iPhones and screen resolutions soon.
